### PR TITLE
SC-121 Modify Put Endpoint for Student-service-api

### PR DESCRIPTION
**Pull request by JOSEEAND**
changes in service.py were bad syntax

### CODE REVIEW

**Review by LilMat69 — CHANGES_REQUESTED** [review]
Please add proper evidence of the method to make sure that runs properly

**Review by ebarahonautn — CHANGES_REQUESTED** [review]
@JOSEEAND please add screnshot that it runs properl.

File: controllers/categorybyId/controller.py
Comment (ebarahonautn) [inline controllers/categorybyId/controller.py:1]: @JOSEEAND 
change this `@auth_required(permission="delete_category", with_args=True)`

by **`@auth_required(permission="delete", with_args=True)`**

**Review by ebarahonautn — CHANGES_REQUESTED** [review]

File: controllers/categorybyId/controller.py
Comment (JOSEEAND) [reply controllers/categorybyId/controller.py:1]: changes ready

**Review by ebarahonautn — APPROVED** [review]

**Pull request merged**